src/lib/piwik.js

```diff
@@ -0,0 +1,126 @@
+/* global __PIWIK_TRACKER_URL__ __PIWIK_SITEID__ __PIWIK_DIMENSION_ID_APP__ */
```

Comment:
We don't have any `.jsx` file in the bar so I didn't want to handle it just for this lib which could be in `.js` in the future. So I wanted to keep the exact same file as in cozy-ui and plan to replace it by the an import from cozy-ui when it will be in `.js` that I think that could be possible.